src/ipynb/spark_ml.ipynb

```diff
@@ -34,7 +34,7 @@
    "source": [
```

Comment:
<p>(If the following is true, please add it)</p><p> </p><p>If you've completed this tutorial as part of the <a href="https://console.cloud.google.com/products/solutions/details/analytics-lakehouse" target="_blank">Analytics Lakehouse</a> solution, you will need to proceed with this step before deleting the solution from your project.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/116/discussion/#comment-1471499167'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>